globals/utilities.php

```diff
@@ -1470,7 +1470,7 @@ function neve_is_new_builder() {
  * @since 3.0.0
  */
 function neve_is_new_skin() {
-	return get_theme_mod( 'neve_new_skin', false ) === 'new';
+	return get_theme_mod( 'neve_new_skin', 'new' ) === 'new';
```

Comment:
@abaicus I think here we should check if neve_new_skin != 'old' since we are setting explicitly the legacy flag